tensorflow/python/data/experimental/ops/threading_options.py

```diff
@@ -52,6 +54,12 @@ class ThreadingOptions(options.OptionsBase):
       "The value 0 can be used to indicate that the threadpool size should be "
       "determined at runtime based on the number of available CPU cores.")
 
+  def _has_non_default_values(self):
+    for attr in filter(lambda opt: not opt.startswith("_"), dir(self)):
+      if getattr(self, attr) is not None:
+        return True
+    return False
+
```

Comment:
remove as this is not used anywhere anymore

Reply:
addressed in the commit